data/SebastianJames55.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "Sebastian James",
+    "bio": "Software developer | Aspiring digital nomad | Open source | x-MECian",
+    "tags": [
+        "Digital nomad",
+        "remote work"
+    ],
+    "socials": [
+        {
+            "icon": "FaInstagram",
+            "url": "https://www.instagram.com/sebastianjames55/"
+        }
+    ],
+    "links": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/call_me_tien",
+            "icon": "FaTwitter"
+        },
+        {
+            "name": "LinkedIn",
+            "url": "https://www.linkedin.com/in/sebastianjames55/",
+            "icon": "FaLinkedIn"
```

Comment:
```diff
- "icon": "FaLinkedIn"
+ "icon": "FaLinkedin"
```

Reply:
Thanks for pointing this out, I missed out this. I'll correct & commit.